lte/gateway/c/oai/include/sgw_context_manager.h

```diff
@@ -38,6 +38,7 @@ void sgw_display_sgw_eps_bearer_context(
     const sgw_eps_bearer_ctxt_t* eps_bearer_ctxt);
 void sgw_display_s11teid2mme(mme_sgw_tunnel_t* mme_sgw_tunnel);
 void sgw_display_s11_bearer_context_information(
+    log_proto_t module,
```

Comment:
minor nit: I'd say it would be better to maintain the order of the added params, `module` param should be the second passed param on this function